tests/relay_integration/lang/java/test_plugin.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+import pytest
+
+from ....sentry.lang.java.test_plugin import BasicResolvingIntegrationTest
```

Comment:
I thought that with `absolute_import` this is not required, but assuming that you tried this out.

Reply:
I remember I had a problem importing this and I "fixed" it with a relative import...  . I'll have another look for a cleaner way to do it and, if I find something, I'll change it.